setup.py

```diff
@@ -0,0 +1,27 @@
+from skbuild import setup
+
+setup(
+    name="rwkv_cpp_python",
```

Comment:
I guess just `rwkv_cpp` or more preferred `rwkv-cpp` is fine, since this is already a Python package :)

Reply:
Sounds good, I'll update this as well. I believe both alias to the same name on PyPI.